GomokuClient/packages/gomoku-core/src/ui/alert-dialog.tsx

```diff
@@ -10,12 +10,21 @@ import {
   CardTitle,
 } from "./card";
 
-interface RematchAlertProps {
+interface AlertDialogProps {
+  title: string;
+  secondaryTitle: string;
+  text: string;
   onAccept: () => void;
   onDecline: () => void;
 }
 
-export const RematchAlert = ({ onAccept, onDecline }: RematchAlertProps) => {
+export const AlertDialog = ({
+  title,
+  secondaryTitle,
+  text,
+  onAccept,
+  onDecline,
+}: AlertDialogProps) => {
```

Comment:
Thank you for making this component more generic! It was on my to-do list, but you took the initiative and solved it. It would also be great if we could make the “Accept” and “Decline” buttons more generic as well, as their text is currently hardcoded

Reply:
Added two more props for button texts